### PR TITLE
COMMERCE-2420 Fix test classes, sf

### PR DESCRIPTION
**Pull request by riccardo-alberti**

### CODE REVIEW

**Comment by riccardo-alberti** [comment]
ci:test

**Comment by ethanbustad** [comment]
Thanks, @riccardo-alberti !

File: commerce-test/src/testIntegration/java/com/liferay/commerce/internal/order/test/CommerceOrderHttpHelperImplTest.java [hunk 1]
```diff
@@ -84,6 +87,20 @@
...
+					getCommerceInventoryBookedQuantities(
+						QueryUtil.ALL_POS, QueryUtil.ALL_POS);
+
+		for (CommerceInventoryBookedQuantity commerceInventoryBookedQuantity :
+				commerceInventoryBookedQuantities) {
+
+			_commerceBookedQuantityLocalService.
+				deleteCommerceInventoryBookedQuantity(
+					commerceInventoryBookedQuantity);
+		}
```
Comment (ethanbustad) [inline commerce-test/src/testIntegration/java/com/liferay/commerce/internal/order/test/CommerceOrderHttpHelperImplTest.java:102]: Oh dang!! This seems like.... kindof a nuclear option. Wouldn't the proper fix be to find wherever we're failing to clean up the `CommerceInventoryBookedQuantity` that a test created, and delete them in the cleanup of that test?

Comment (ethanbustad) [reply commerce-test/src/testIntegration/java/com/liferay/commerce/internal/order/test/CommerceOrderHttpHelperImplTest.java:102]: @marco-leo I'm ok with doing it this way for now if you are. But it'd be nice to switch it to the proper fix after we release :)

Comment (marco-leo) [reply commerce-test/src/testIntegration/java/com/liferay/commerce/internal/order/test/CommerceOrderHttpHelperImplTest.java:102]: Agree! @riccardo-alberti  please send a fix when you get time. Thanks

**Comment by marco-leo** [comment]
Thanks @ethanbustad @riccardo-alberti @petershin !!!

**Pull request merged**